matchbox_demo/src/main.rs

```diff
@@ -34,9 +38,9 @@ fn main() {
         // define system that returns inputs given a player handle, so GGRS can send the inputs around
         .with_input_system(input)
         // register types of components AND resources you want to be rolled back
-        .register_rollback_type::<Transform>()
-        .register_rollback_type::<Velocity>()
-        .register_rollback_type::<FrameCount>()
+        .register_rollback_component::<Transform>()
+        .register_rollback_component::<Velocity>()
+        .register_rollback_resource::<FrameCount>()
```

Comment:
Could you revert this? I don't like the new perspective/lighting

Reply:
I don't understand which change you're referring to? `register_rollback_type` doesn't exist as a method anymore

Reply:
Nevermind, I found the changes you meant